src/ScraperType/CourseTopicScraper/ScraperModules/ApiUtility.py

```diff
@@ -76,19 +76,32 @@ def getCourseCollectionsJson(self, courseApiUrl, courseUrl):
             topicNameList = []
             baseApiUrl = f"https://educative.io/api/collection/{authorId}/{collectionId}/page/"
             categoryType = ["COLLECTION_PROJECT", "COLLECTION_CATEGORY", "COLLECTION_ASSESSMENT", "PATH_EXTERNAL_PROJECT", "PATH_EXTERNAL_ASSESSMENT", "CLOUD_LAB"]
+            idx = 0
+            toc = []
             for category in categories:
                 if any(cType in category["type"] for cType in categoryType) and (
                         isinstance(category["id"], int) or len(category["id"]) <= 10):
                     if not category["pages"]:
                         topicApiUrlList.append(baseApiUrl + str(category["id"]) + f"?work_type={courseType}")
```

Comment:
@anhpho can you create a variable and store te api url, and then use the variable where it is required. looks cleaner